web_drop_target/static/src/js/web_drop_target.js

```diff
@@ -140,7 +140,15 @@ odoo.define("web_drop_target", function(require) {
          */
         _onBodyFileDragover: function(ev) {
             ev.preventDefault();
-            if (_.isEmpty(this._get_drop_items(ev))) {
+            const actionManager = this.findAncestor(function(ancestor) {
+                return ancestor instanceof ActionManager;
+            });
+            const controller = actionManager.currentDialogController;
+            if (
+                _.isEmpty(this._get_drop_items(ev)) &&
+                this.renderer.chatter &&
```

Comment:
Won't this make it not work if you don't have a chatter? for example, DMS.

Reply:
Ouch!!! Thanks, I think I have a better idea now :wink: